next/components/forms/signer/useFormSignerLoader.tsx

```diff
@@ -58,13 +71,21 @@ export const FormSignerLoaderProvider = ({ children }: PropsWithChildren) => {
   const handleScriptsLoaded = () => {
     ditec.dSigXadesBpJs.detectSupportedPlatforms(null, {
       onSuccess: (result) => {
```

Comment:
After the signer is loaded it outputs `platforms` array that you need to pass back to `deploy` function when starting the signer. This wasn't needed in the previous version.